apps/platform-app/src/app/[locale]/(auth)/(routes)/signout/page.tsx

```diff
@@ -0,0 +1,14 @@
+'use client'
```

Comment:
**nit:** no need to make it client side, let's make this sign out it server side, so redirect happens there